# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/CookieStore.h
# Commit: 0d57b435d872
# Full Hash: 0d57b435d872751c2a26879fb04bf8745314e37a
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r a63a5bcb0524 -r 0d57b435d872 dom/cookiestore/CookieStore.h
--- a/dom/cookiestore/CookieStore.h	Wed Sep 11 16:26:57 2024 +0000
+++ b/dom/cookiestore/CookieStore.h	Wed Sep 11 16:26:57 2024 +0000
@@ -17,6 +17,7 @@
 
 class CookieData;
 class CookieStoreChild;
+class CookieStoreNotificationWatcherWrapper;
 class CookieStoreNotifier;
 class Promise;
 
@@ -72,6 +73,7 @@
 
   RefPtr<CookieStoreChild> mActor;
   RefPtr<CookieStoreNotifier> mNotifier;
+  RefPtr<CookieStoreNotificationWatcherWrapper> mNotificationWatcher;
 };
 
 }  // namespace mozilla::dom